go/vt/vttablet/tabletserver/health_streamer_test.go

```diff
@@ -404,6 +404,10 @@ func TestReloadView(t *testing.T) {
 	db.AddQuery("SELECT TABLE_NAME, CREATE_TIME FROM _vt.`tables`", &sqltypes.Result{})
 	// adding query pattern for udfs
 	db.AddQueryPattern("SELECT name.*", &sqltypes.Result{})
+	db.AddQuery("select table_name, partition_name from information_schema.partitions where table_schema = database() and partition_name is not null", &sqltypes.Result{})
```

Comment:
I don't love that these queries have leaked out of the `schema` package, but I can't think of better way to make this test pass (unless we want to make the query constants public. Interested if this causes anyone concern.